public/profile.css

```
@@ -0,0 +1,72 @@
+@charset "utf-8";
+
+body {
+  display: flex;
+  font-family: sans-serif;
+  justify-content: center;
+  margin:0;
```

```suggestion
  margin: 0;
```
スペース欲しいです！
他でも同じことになっているので合わせて修正してくださいー！